src/os/bluestore/aio.cc

```diff
@@ -1,9 +1,19 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
+#include "acconfig.h"
+#include "include/compat.h"
```

Comment:
might want to move these two includes after `#include <algorithm>`.

Reply:
@tchaikov 
I will, but can you explain why?

Because I advise against including (boost) C++ files before `include/compat.h` since it will interfere with the definition of ENODATA.

Reply:
see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

yeah, but i think `<algorithm>` does not fall into that bucket.